src/DoctrineWriter.php

```
@@ -262,47 +287,50 @@ protected function disableLogging()
      */
     protected function reEnableLogging()
     {
-        $config = $this->entityManager->getConnection()->getConfiguration();
+        //TODO: do we need to add support for MongoDB logging?
+        if (!($this->objectManager instanceof \Doctrine\ORM\EntityManager)) return;
```

Should throw an exception if the odm is used.